miner/worker.go

```diff
@@ -74,6 +74,9 @@ const (
 
 	// staleThreshold is the maximum depth of the acceptable stale block.
 	staleThreshold = 7
+
+	maxClockSkewSeconds   = 600 // 10 mins for now
```

Comment:
A nice to have would be using `time.Minute` like so:

```go
maxClockSkewSeconds = 10 * time.Minute
```

You above the same pattern is followed using `time.Second`

Reply:
Ahh yeah, those end up being durations which are denominated in nanos, but I'd like actual seconds. I guess I could do `10 * time.Minute / time.Nanosecond` but I think 600 = 10 mins in seconds could be a bit more straight-forward.